Dockerfile

```diff
@@ -60,5 +60,4 @@ FROM $BASE_IMAGE AS subtensor
 
 COPY --from=builder /subtensor/snapshot.json /
 COPY --from=builder /subtensor/raw_spec.json /
```

Comment:
isn't this removed as well?

Reply:
Here as well:

https://github.com/opentensor/subtensor/blob/e8eae823ebf581bc8b90498eaa22bbdae976701f/scripts/run/subtensor.sh#L45